src/main/java/in/ac/bits/protocolanalyzer/persistence/repository/SaveRepository.java

```diff
@@ -144,4 +146,36 @@ private void publishEndOfSave(long time) {
 		//log.info("Publishing end of Save Repository");
 		eventBus.post(new SaveRepoEndEvent(time));
 	}
-}
+
+	public int getLowWaterMark() {
+		return lowWaterMark;
+	}
+
+	public void setLowWaterMark(int lowWaterMark) {
+		this.lowWaterMark = lowWaterMark;
+	}
+
+	public boolean isAnalysisOnly() {
+		return analysisOnly;
+	}
+
+	public void setAnalysisOnly(boolean analysisOnly) {
+		this.analysisOnly = analysisOnly;
+	}
+
+	public ConcurrentLinkedQueue<ArrayList<IndexQuery>> getBuckets() {
+		return buckets;
+	}
+
+	public void setBuckets(ConcurrentLinkedQueue<ArrayList<IndexQuery>> buckets) {
+		this.buckets = buckets;
+	}
+
+	public boolean isAnalysisRunning() {
```

Comment:
nothing interesting in this method. Please remove the same and let the boolean variable be accessible.